src/bun.js/bindings/webcore/FetchHeaders.cpp

```diff
@@ -261,30 +262,58 @@ void FetchHeaders::filterAndFill(const HTTPHeaderMap& headers, Guard guard)
     }
 }
 
-static NeverDestroyed<const String> setCookieLowercaseString(MAKE_STATIC_STRING_IMPL("set-cookie"));
+static const WTF::String setCookieLowercaseString = WTF::staticHeaderNames[static_cast<uint8_t>(HTTPHeaderName::SetCookie)];
+
+static bool compareIteratorKeys(const String& a, const String& b)
```

Comment:
this can be removed since we no longer useit here

Reply:
updated